### ISSUE TITLE
Allure property is undefined when process.env is assigned to config.env before allureWriter(on, config);

### ISSUE THREAD

**Issue by romankhomitskyi** [Shelex/cypress-allure-plugin#110]
**Describe the bug**
In my plugins/index.ts
I have the following code 
```
config.env = process.env;
allureWriter(on, config);

return config;
```

Then I pass the following arguments in CLI

`yarn cypress open --env allure=true,configFile=local,allureLogCypress=true`

As you can see in configuration allure is undefined. I am not sure whether it's a bug, but why only this property gets overridden?

<img width="221" alt="Screenshot 2021-12-08 at 21 03 12" src="https://user-images.githubusercontent.com/57874507/145268195-dc238766-027a-42f7-a4ff-839c30374f46.png">

When I assign process.env to other property

```
config.env.devEnv = process.env;
allureWriter(on, config);

return config;
```
<img width="238" alt="Screenshot 2021-12-08 at 21 13 32" src="https://user-images.githubusercontent.com/57874507/145269546-d5e19a50-4322-4a10-a274-2b7118fab109.png">

Then there is **another** allure option under **devEnv** at the end
<img width="315" alt="Screenshot 2021-12-08 at 21 14 00" src="https://user-images.githubusercontent.com/57874507/145269619-dc699be4-74b0-4c0d-ab78-4d3899d94861.png">

**Environment (please complete the following information):**
- Cypress version:   8.6.0
- Allure: 2.22.0

**Comment by Shelex** [comment]
Hi @romankhomitskyi
Issue is fixed in v2.24.0.

Problem is that `process.env` was previously used to pass some allure configuration, so process.env.allure is overwritten by plugin also. Now it is removed as obsolete.

**Comment by romankhomitskyi** [comment]
Thanks for fix
